### DISCUSSION TITLE
Infinitime will not boot after the battery fully dies

### DISCUSSION THREAD

**Discussion by adryd325** [InfiniTimeOrg/InfiniTime#1674] [Q&A]
### Verification

- [X] I searched for similar bug reports and found none was relevant.

### What happened?

I left my watch off for a week and it won't boot

### What should happen instead?

It should boot

### Reproduction steps

Leave the watch's battery to die

Attempt to power on after a long time

### More details?

I tried holding the side button to reboot it, holding it until the pine symbol turned blue and held it till it turned red and none of these worked

### Version

v1.11.0

**Comment by ajack2001my** [comment]
Leave it on the charger for a few day, it will eventually start-up, then keeps rebooting until there is enough charge in battery to allow a "normal" boot. That will then let it charge till full.

**Comment by adryd325** [comment]
it's stuck at the infinitime slash screen for hours

**Reply by JF002** [reply]
The InfiniTime logo shows that the watch rebooted in the Recovery firmware (more info [here](https://github.com/InfiniTimeOrg/InfiniTime/blob/main/doc/gettingStarted/about-software.md) and [here](https://github.com/InfiniTimeOrg/pinetime-mcuboot-bootloader#recovery-firmware). 

This firmware allows to install (OTA - over the air) a new firmware using a companion app and BLE. Once the firmware installation is done, your watch should reboot in InfiniTime as expected.

**Reply by sammyj85** [reply]
I'm having the same issue, but I'm struggling to recover.

Are there any particular tips of what works to recover?

If I hold the button to go to the red pinecone on boot, it does a flicker, then starts the green pinecone again.

If I hold the button to go to the blue pinecone on boot, it does a flicker, then starts the green pinecone again.

If I let the green pinecone go, it pauses at 98% green for 10 seconds, then shows the white Infinitime logo.

I tried connecting with Watchmate, and it fails: "Device is rejected by the app".

I can connect with Siglo. Attempting to upload 1.12.0 or 1.11.0 pinetime-mcuboot-app-dfu used to pause forever at 0 bytes transferred. I killed the process (maybe that was bad?) to try something else after a few mins, and now Siglo always says "The firmware update has failed. Retrying the update will fix it in most cases." It doesn't. I can restart both the watch and the app freshly and try again, but nothing changes. Attempting to update the infinitime-resources also freezes at 0 bytes.

Are there any tried and true methods for updating with the recovery firmware?

**Reply by sammyj85** [reply]
Ah, brilliant, Gadgetbridge worked! Not sure what was up with the other two.
